packages/devtools/web/styles.css

```diff
@@ -520,7 +520,7 @@ div.tabnav {
 .frame-timeline {
     position: relative;
     margin-top: 4px;
-    whitespace: pre;
+    white-space: pre;
```

Comment:
DBC: I don't think this white-space property is necessary given the (new) one below.

Reply:
done.